### ISSUE TITLE
Not really a feature request, just appreciation

### ISSUE THREAD

**Issue by Daksol** [ronivay/XenOrchestraInstallerUpdater#224]
Just wanted to express appreciation and thanks for the efforts and commitment over time by @ronivay in maintaining this excellent installer. Something good was set up, and then maintained well over time, been responsive to users (even those with dumb questions) - which is not something all projects achieve.

**Comment by vmpr** [comment]
yes, Daksol is right, what a great project! thank you from Australia. Good Job :)

**Comment by osuno1** [comment]
I also totally agree!  I use his work almost monthly to keep things running and up to date. I really appreciate all your work and effort!

**Comment by rizaemet** [comment]
I also want to thank you for your efforts @ronivay. Thanks to you, we always stay up to date and this happens smoothly.

**Comment by kismetgerald** [comment]
Wanted to add my voice to this as well.  Greatly appreciate this project.  For a Windows System Administrator, with sufficient knowledge to be "dangerous" in the Linux world, this project has definitely been a game changer in managing an XCP-ng environment.

**Comment by MrGrymReaper** [comment]
Thank you for all you effort with this script. Especially with the VM download for importing directly into XCP-ng or XenServer.

**Comment by gusosborne** [comment]
Yes, thank you so much for putting this out there!

**Comment by bogdantomasciuc** [comment]
Indeed! Thank you for all you effort!

**Comment by shepard1532** [comment]
I would also like to thank you @ronivay. It's a great project helping to maintain a XCP-ng environment. Greetings from Switzerland :)

**Comment by Split-Infinitive** [comment]
Many thanks @ronivay.
Your efforts are greatly appreciated.
Greetings from Adelaide, South Australia

**Comment by Vertux** [comment]
@ronivay 
I would like to join in with all the thanks. It is just a great project that helps me a lot to maintain my little xcp-ng homelab and makes me happy. Many, many thanks for all your work from Germany.

**Comment by aethne0** [comment]
extremely sick my friend

**Comment by chrispro-21** [comment]
@ronivay You have saved me hours and hours of work with this. I routinely recommend people over to your repo because of the sheer awesome work you have done here. Thank you so much!

**Comment by LoganRocks** [comment]
Really appreciate the work you put into this script. Happy new year @ronivay
